scripts/publish-staged

```diff
@@ -33,6 +33,7 @@ usage() {
 	echo "Options"
 	echo "  -d  true|false  Dryrun (default is true)"
 	echo "  -p  PROFILE     AWS CLI Profile (default is none)"
+	echo "  -o  PROFILE     AWS CLI Profile for the bucket to push to (defaults to the profile specified with -p)"
```

Comment:
Why is this being introduced? Is it mostly to enforce stricter profiling logic? If `-p` will be used if this not specified anyway, why do we even need this flag?